www/front_src/src/Resources/Graph/Performance/Graph/index.tsx

```diff
@@ -560,6 +560,7 @@ const GraphContent = ({
             />
           </Group.Group>
           <TimeShiftContext.Provider
+            // eslint-disable-next-line react/jsx-no-constructed-context-values
```

Comment:
As suggested by the [rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-constructed-context-values.md), we should wrap the context values inside a useMemo